controller/src/main/java/org/jboss/as/controller/logging/ControllerLogger.java

```diff
@@ -3209,9 +3209,23 @@ XMLStreamException discoveryOptionsMustBeDeclared(String adminOnlyCmd, String po
     @Message(id = 364, value = "Capability '%s' is unknown.")
     IllegalStateException unknownCapability(String capability);
 
+    @Message(id = NONE, value = "It was not possible to get satisfactory capability for capability '%s' even when " +
```

Comment:
This would not be NONE; it's a different message for a different call path and should have different id. Same for the others.